tools/src/main/java/org/neo4j/tools/dump/DumpStore.java

```diff
@@ -58,6 +60,29 @@
  */
 public class DumpStore<RECORD extends AbstractBaseRecord, STORE extends RecordStore<RECORD>>
 {
+    private static class IdRange
```

Comment:
would be nice to get java doc that explains format in which idRange should be supplied